Project-Hotpot/Project-Hotpot/Managers/SpotifyManager.swift

```diff
@@ -21,15 +21,17 @@ class SpotifyManager: NSObject {
                     NSLog("Fetching token request error \(error)")
                     return
                 }
-                let accessToken = dictionary!["access_token"] as! String
+                guard let accessToken = dictionary?["access_token"] as? String
+                else {
```

Comment:
Another case here where we typically put this else at the end of the `guard` line.